README.md

```diff
@@ -32,6 +32,7 @@ Then configure the rules you want to use under the rules section.
     "jest/no-disabled-tests": "warn",
     "jest/no-focused-tests": "error",
     "jest/no-identical-title": "error",
+    "jest/prefer-to-be-null": "warn",
```

Comment:
IMO this should be off by default, otherwise it's a breaking change

Reply:
Why so?
Same for all others?

Reply:
If I use `toBe(null)` somewhere in my code, and have activated the recommended rules, my build is potentially broken in a feature release (or we can do a major, but I don't think that's necessary).

other open PRs yes

Reply:
It's a "warn" level, so by default it should not fail the lint command.

Reply:
Just removed all warnings. 😓 
Should redo?

Reply:
At work, we have `--no-warnings`

Reply:
How about moving forward on this at the moment?

Reply:
Oh, sorry! yeah, let's just roll with it